api/apps/api/src/modules/access-control/scenarios-acl/scenario-acl.controller.ts

```diff
@@ -0,0 +1,54 @@
+import { apiGlobalPrefixes } from '@marxan-api/api.config';
+import {
+  Controller,
+  Req,
+  Param,
+  ParseUUIDPipe,
+  UseGuards,
+  ForbiddenException,
+  HttpCode,
+  HttpStatus,
+  Delete,
+} from '@nestjs/common';
+import { JwtAuthGuard } from '@marxan-api/guards/jwt-auth.guard';
+import { ScenarioAclService } from '@marxan-api/modules/access-control/scenarios-acl/scenario-acl.service';
+import { RequestWithAuthenticatedUser } from '@marxan-api/app.controller';
+import {
+  ApiBearerAuth,
+  ApiNoContentResponse,
+  ApiOperation,
+  ApiTags,
+} from '@nestjs/swagger';
+import { isLeft } from 'fp-ts/lib/These';
+
+@UseGuards(JwtAuthGuard)
+@ApiBearerAuth()
+@ApiTags('Scenarios-Users Roles')
+@Controller(`${apiGlobalPrefixes.v1}/roles/scenarios`)
+export class ScenarioAclController {
+  constructor(private readonly scenarioAclService: ScenarioAclService) {}
+
+
+  @Delete(':scenarioId/users/:userId')
+  @HttpCode(HttpStatus.NO_CONTENT)
+  @ApiOperation({ summary: 'Revoke access to user from project' })
+  @ApiNoContentResponse({
+    status: 204,
+    description: 'User was deleted correctly',
+  })
+  async deleteUserFromProject(
+    @Param('scenarioId', ParseUUIDPipe) scenarioId: string,
+    @Param('userId', ParseUUIDPipe) userId: string,
+    @Req() req: RequestWithAuthenticatedUser,
+  ): Promise<void | boolean> {
+    const result = await this.scenarioAclService.revokeAccess(
+      scenarioId,
+      userId,
+      req.user.id,
```

Comment:
user id twice may be misleading, consider slight rename to diff from requester/subject